client/my-sites/domains/domain-management/dns/dns-records-list-item.jsx

```diff
@@ -19,7 +19,7 @@ function DnsRecordsListItem( { type, name, value, actions, disabled, isHeader, r
 			position="bottom left"
 		>
 			{ actions.map( ( action ) => {
-				return (
+				return action.disabled ? null : (
```

Comment:
If we remove the `disabled` property, then I guess we don't need this conditional here either.